web/source/dom/domEventHandlers.ts

```diff
@@ -197,6 +197,15 @@ namespace com.keyman.dom {
         }
       }
 
+      if(DOMEventHandlers.states._IgnoreNextSelChange) {
+        // If a keyboard calls saveFocus() (KSF), then ignore the
```

Comment:
Where's the check for `KSF`?

Reply:
`_IgnoreNextSelChange` is the value set by `KSF()`/`saveFocus()`.